project.mml

```diff
@@ -816,8 +816,9 @@ Layer:
             name
           FROM planet_osm_line
           WHERE aerialway IS NOT NULL
-            OR (man_made IN ('goods_conveyor', 'pipeline')
+            OR (man_made = 'pipeline'
                 AND tags-> 'location' IN ('overground', 'overhead', 'surface', 'outdoor')
+                OR man_made = 'goods_conveyor'
```

Comment:
This line needs to move down below "OR bridge IN...", outside of the close parenthesis .

Also, I see that there are 217 man_made=goods_conveyor ways with `tunnel=yes` or `location=underground`. It would be good to exclude these from rendering.

Reply:
fixed